sdk/spring/azure-spring-boot-test-selenium-common/src/main/java/com/azure/test/aad/common/SeleniumITHelper.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.test.aad.common;
+
+import com.azure.spring.test.AppRunner;
+import java.io.File;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeDriverService;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class SeleniumITHelper {
+    protected AppRunner app;
+    protected WebDriver driver;
+    protected WebDriverWait wait;
+    protected static Map<String, String> DEFAULT_PROPERTIES = new HashMap<>();
```

Comment:
How about delete `DEFAULT_PROPERTIES` and provide method in SeleniumITHelper's **sub** class:
```
public static Map<String, String> createDefaultProperteis(){...}
```
Which will create a new map on every execution. Then the user can decide use `createDefaultProperteis` or not.